src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewRenderer.php

```diff
@@ -115,8 +115,8 @@ public function render(
         $partial = false,
         $options = []
     ) {
-        $webspaceKey = $options['webspaceKey'];
-        $locale = $options['locale'];
+        $webspaceKey = $options['webspaceKey'] ?? null;
+        $locale = $options['locale'] ?? null;
```

Comment:
Where this maybe make sense in point of view from the `admin` this doesn't reflect how sulu would behave on the website where a webspace key and locale is always required.

Reply:
yeah thats right.. but i think it should not hurt to add a fallback here and we do the same thing in the `Preview` class 🙂 
https://github.com/sulu/sulu/blob/5e4f11ac8d160c4a1242b8454265925c72a9cf30/src/Sulu/Bundle/PreviewBundle/Preview/Preview.php#L95

but i dont have a strong opinion, is also okay for me to close this